OpenRA.Game/Network/Order.cs

```diff
@@ -235,10 +234,6 @@ public Order(string orderString, Actor subject, bool queued)
 		public Order(string orderString, Actor subject, Target target, bool queued)
 			: this(orderString, subject, target, null, queued, CPos.Zero, 0) { }
 
-		public Order(string orderstring, Order order)
-			: this(orderstring, order.Subject, order.Target,
-				   order.TargetString, order.Queued, order.ExtraLocation, order.ExtraData) { }
-
```

Comment:
This was unused - figured safest to dispose of - left me know if it should be kept.